flake.nix

```diff
@@ -16,9 +16,8 @@
   };
 
   outputs = { self, nixpkgs, utils, ... }@inputs:
-  rec {
-    overlay = final: prev: let
```

Comment:
Can we preserve the `overlay` output for the sake of backward compatibility?
Just leave it as `overlay = overlays.default`